Add custom transport sample (#2567)
wso2/docs-ei: en/micro-integrator/docs/setup/transport_configurations/configuring-transports.md

listener.enable = false
sender.enable =false
```
## Configuring custom transports

> Other than the transports defined above, you can use custom transport that let you to add new transport to the MI. Custom transport configuration contain senders and listeners that you can define a transport as follows.

Reviewer: ```suggestion
Other than the transports defined above, you can use a custom transport that enables you to add a new transport to the Micro Integrator. Custom transport configurations contain senders and listeners that you can define. A custom transport configuration is as follows.
```

```toml tab='Custom Listener'
[[custom_transport.listener]]
class="org.wso2.micro.integrator.TransportListener"
protocol = "protocol"
```

```toml tab='Custom Sender'
[[custom_transport.sender]]
class="org.wso2.micro.integrator.TransportSender"
protocol = "protocol"
parameter.'transport.param' = true
```

> You can define the listener and sender classes in `class` parameter. Any additional parameter can be pass to the configuration with `parameter`. Check the following usage of custom transport to configure HL7 transport.

Reviewer: ```suggestion
You can define the listener and sender classes in the `class` parameter. Any additional parameter can be passed to the configuration using `parameter`. The following section involves using a custom transport to configure HL7 transport.
```

## Configuring the HL7 transport

The HL7 transport allows you to handle Health Level 7 International (HL7) messages. 

### Adding the transport

HL7 is not shipped by default in the pack. To make the transport available, download the [HL7_ZIP](https://github

> .com/wso2-docs/WSO2_EI/raw/master/micro-integrator-resources/hl7/1.2.0/wso2mi-hl7-1.2.0.zip), extract it and add both

Reviewer: ```suggestion
.com/wso2-docs/WSO2_EI/raw/master/micro-integrator-resources/hl7/1.2.0/wso2mi-hl7-1.2.0.zip), extract it, and add both
```

>  the jars inside it to the `<MI_HOME>/dropins` folder.

Reviewer: ```suggestion
 the .jar files inside it to the `<MI_HOME>/dropins` folder.
```

### Enabling the transport

> Add the following configurations to the `deployment.toml` file (stored in the `<MI_HOME>/conf` folder: 

Reviewer: ```suggestion
Add the following configurations to the `deployment.toml` file (stored in the `<MI_HOME>/conf` folder): 
```
